course_contents/1_intro/notes/05. Logical Operators.md

```diff
@@ -0,0 +1,35 @@
+## AND
+- Returns True if both operands are true
+
+Example:
+
+| Statement         | Result     |
+| -------------     | ---------- |
+| `True and True`   | `True`     |
+| `True and False`  | `False`    |
+| `False and False` | `False`    |
+| `False and True`  | `False`    |
+
+
+## OR
+- Returns True if at least one operand is True
+- Reads from the left to the right operand
+
+Example:
+
+| Statement         | Result     |
+| -------------     | ---------- |
+| `True or True`    | `True`     |
+| `True or False`   | `True`     |
+| `False or False`  | `False`    |
+| `False or True`   | `True`     |
```

Comment:
I'd change this to be more technically accurate, even if it's a bit more complicated.

`and` returns the first value if it is Falsy, or the second value otherwise.
`or` returns the first value if it is Truthy, or the second value otherwise.

Might have to explain Truthy / Falsy around here in another short segment.